go.test.sh

```diff
@@ -1,12 +1,18 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
 set -e
-echo "" > coverage.txt
+#set -x # debug/verbose execution
+
+# Ensures removal of file
+trap 'rm -f profile.out downloader/*.{mp4,download,jpg}' SIGINT EXIT
+
+# go -race needs CGO_ENABLED to proceed
+export CGO_ENABLED=1;
 
 for d in $(go list ./... | grep -v vendor); do
-    go test -race -coverprofile=profile.out -covermode=atomic "$d"
-    if [ -f profile.out ]; then
-        cat profile.out >> coverage.txt
-        rm profile.out
-    fi
+	go test -v -timeout=2m -race -coverprofile=profile.out -covermode=atomic "${d}"
+	if [ -f profile.out ]; then
+		cat profile.out >coverage.txt
```

Comment:
`cat profile.out > coverage.txt`

Reply:
instead of removing the coverage.txt at the very first of it all, it does overwrite it when profile.out has been generated. This way, coverage generated from the last run is not affected if the curent run fails

Reply:
You misunderstood, I mean a space needed before `coverage.txt` (format issue